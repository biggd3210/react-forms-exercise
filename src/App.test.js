import { render, screen } from '@testing-library/react';
import App from './App';

test('renders learn react link', () => {
  render(<App />);
});

test('matches snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});