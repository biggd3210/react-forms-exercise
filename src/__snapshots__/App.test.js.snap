// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <div>
        <form>
          <label
            for="height"
          >
            Height
          </label>
          <input
            id="height"
            name="height"
            type="number"
            value="0"
          />
          <label
            for="width"
          >
            Width
          </label>
          <input
            id="width"
            name="width"
            type="number"
            value="0"
          />
          <label
            for="color"
          >
            Color
          </label>
          <input
            id="color"
            name="color"
            type="text"
            value=""
          />
          <button
            type="submit"
          >
            Add a new box!
          </button>
        </form>
      </div>
      <div />
    </div>
  </div>
</DocumentFragment>
`;
